scripts/grub.cfg

```diff
@@ -71,12 +71,13 @@ play 960 440 1 0 4 440 1
 menuentry --hotkey=i --id=video 'Start $vendor Installation' {
     load_video
     set background_color=black
-    linux    /vmlinuz gfxpayload=text quiet nomodeset boot=live toram=filesystem.squashfs $gfx_console
+    linux    /vmlinuz gfxpayload=text quiet boot=live toram=filesystem.squashfs $gfx_console
```

Comment:
Why are we removing nomodeset?

Reply:
It is a change we do not want to merge and was one of the things done while debugging issues related to grub - testing without this (change has been reverted)